src/games/strategy/engine/framework/GameRunner2.java

```diff
@@ -151,11 +158,18 @@ private static void usage() {
         + "   to start a server, you can optionally password protect the game using triplea.server.password=foo");
   }
 
-  public static void main(final String[] args) {
+  public static void main(String[] args) {
+    if( args == null ) {
```

Comment:
I would prefer to annotate it @NonNull instead, make it part of the contact that it cannot be null, call it correclty